### PR TITLE
PRS13840 44

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New records for Abb 199 and 168 .

### CODE REVIEW

File: new/PRS13840Yamesrac.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                    <persName type="alt"><roleName type="title">ʾabbā</roleName> Yamǝsrač</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic"/>
+                    <nationality type="Ethiopia"/>
+                    <birth/>
+                    <death/>
+                    <floruit notBefore="1770" notAfter="1860"/>
+                </person>
+                <listRelation>
+                    <relation name="saws:isRelatedTo" active="PRS13840Yamesrac" passive="BNFabb199"/>
```
Comment (thea-m) [inline new/PRS13840Yamesrac.xml:59]: This relation should only be used between works. maybe lawd:hasAttestation and syriaca:has-relation-to-place would work?

**Review by thea-m — CHANGES_REQUESTED** [review]

File: new/PRS13840Yamesrac.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/PRS13840Yamesrac.xml:59]: ok, changed.

**Review by thea-m — CHANGES_REQUESTED** [review]

**Comment by DenisNosnitsin1970** [comment]
Sorry, what do you mean?

**Comment by DenisNosnitsin1970** [comment]
Yes, has Attestation.

**Pull request merged**